feature-sessions/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
     <!-- Session Detail Screen -->
     <string name="session_description_read_more_text">続きを見る</string>
+    <string name="session_schedule">%1$s %2$s %3$s-%4$s</string>
```

Comment:
I think you can add 月 日 here. I think if you do this, you don't need if expression in the compose. 🙏

Reply:
Are you ok an expression in English as bellow?

```
10 5th 10:00-11:00
```

I used if expression for capitalized month name in English.

```
October 5th 10:00-11:00
```

Reply:
I see probably I made a mistake about this. Probably we mustn't use strings.xml. Because we will have to manage two place.
Can you remove it from strings.xml and make date string in Kotlin?

Reply:
> Can you remove it from strings.xml and make date string in Kotlin?

OK. I will try it!

Reply:
@takahirom 
Fixed!
Please check when you have the time. 🙏🏼